tests/test_json_schema_test_suits.py

```diff
@@ -2,17 +2,36 @@
 from pathlib import Path
 
 import pytest
+import requests
 
-from fastjsonschema import CodeGenerator, JsonSchemaException, compile
+from fastjsonschema import CodeGenerator, RefResolver, JsonSchemaException, compile
 
+remotes = {
+    'http://localhost:1234/integer.json': {'type': 'integer'},
+    'http://localhost:1234/name.json': {
+        'type': 'string',
+        'definitions': {
+            'orNull': {'anyOf': [{'type': 'null'}, {'$ref': '#'}]},
+        },
+    },
+    'http://localhost:1234/subSchemas.json': {
+        'integer': {'type': 'integer'},
+        'refToInteger': {'$ref': '#/integer'},
+    },
+    'http://localhost:1234/folder/folderInteger.json': {'type': 'integer'}
+}
+def remotes_handler(uri):
+    print(uri)
+    if uri in remotes:
+        return remotes[uri]
+    return requests.get(uri).json()
 
 def pytest_generate_tests(metafunc):
     suite_dir = 'JSON-Schema-Test-Suite/tests/draft4'
     ignored_suite_files = [
-        'definitions.json',
         'ecmascript-regex.json',
-        'ref.json',
-        'refRemote.json',
+    ]
+    ignore_tests = [
```

Comment:
Probably this could be removed as it's not needed in the end.

Reply:
Maybe we should keep that, because there might be new tests in JSON-schema-test-suite in future that might not pass. Even it is useless now?
Also ignoring just tests might help, when implementing dfaft6, draft7....

Reply:
Ok, why not.